lib/components/user/existing-account-display.tsx

```diff
@@ -48,6 +49,13 @@ const ExistingAccountDisplay = (props: Props) => {
         <FormattedMessage id="components.MobilityProfile.MobilityPane.header" />
       )
     },
+    {
+      collapsible: true,
```

Comment:
I'm not the biggest fan of it here. I think this is a nice to have feature, but it doesn't make sense if this is the only section that uses it. If all the sections were collapsable I think this would make more sense.

Reply:
Agreed